src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewRoundedCell.swift

```diff
@@ -0,0 +1,97 @@
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import UIKit
+
+class DynamicTableViewRoundedCell: UITableViewCell {
+
+	// MARK: - View elements.
+
+	lazy var title = UILabel(frame: .zero)
+	lazy var body = UILabel(frame: .zero)
```

Comment:
I think that we have a dedicated label for our predefined styles. However, it seems `ENALabel` cannot be used from code - only via IB. 😭 @EisvogelSAP do you have any insight into this?

Because of the fact that `ENALabel` does not seem to be usable from code I will approve this but maybe we should take the time once we are feature complete to make `ENALabel` code compatible...

Reply:
Why wouldn't `ENALabel` be usable in Code?